metrics-operator/chart/templates/metrics-validating-webhook-configuration.yaml

```diff
@@ -3,7 +3,9 @@ kind: ValidatingWebhookConfiguration
 metadata:
   name: metrics-validating-webhook-configuration
   annotations:
```

Comment:
this correction is not good, if the global value does not exist we template an invalid yaml with empty annotations

Reply:
You could add annotations inside the if but this won't work after we merge https://github.com/keptn/lifecycle-toolkit/pull/2770